Updated MS investment landing page (#7927)
mattermost/docs: source/about/maximize-microsoft-investments.rst

The following mission-ready collaboration capabilities are available:

On-Premises Skype for Business Replacement
-------------------------------------------

As Skype for Business reaches end-of-life, secure organizations require an alternative that preserves on-premises control, integrates into Microsoft workflows, and meets the security standards of air-gapped and classified environments. Cloud-first replacements like Microsoft Teams are not always viable due to network segmentation, compliance restrictions, or data sovereignty mandates.

- **Preserve mission-critical communication workflows** with a self-hosted Mattermost deployment that supports :doc:`1:1 calls </collaborate/make-calls>`, :ref:`screen sharing <collaborate/make-calls:share your screen>`, and :doc:`threaded messaging </collaborate/organize-conversations>` within secure environments. See :doc:`Mattermost for M365, Teams, and Outlook </integrate/mattermost-mission-collaboration-for-m365>`.

> - **Integrate Mattermost with Microsoft tools** such as Outlook, Teams, and :doc:`Entra ID Single Sign-On </onboard/sso-entraid>` to retain user workflows while centralizing identity and access control.

Reviewer: Should the "mission collab for m365 go in this line instead? The first bullet is more about channel based messaging, where this is about integration into suite of MS tools.

> - **Deploy in sovereign, air-gapped, or private cloud environments** such as `Azure Deployment <https://azuremarketplace.microsoft.com/en-us/marketplace/apps/mattermost.mattermost-operator?tab=overview>`_ to maintain compliance with STIG, FedRAMP, and NIST 800-53 standards.

Reviewer: ```suggestion
- **Deploy in sovereign, air-gapped, or private cloud environments** such as `Azure Deployment <https://azuremarketplace.microsoft.com/en-us/marketplace/apps/mattermost.mattermost-operator?tab=overview>`_ while maintaining compliance with STIG, FedRAMP, and NIST 800-53 standards.
```

:doc:`Learn more </about/on-prem-skype-for-business-replacement>` about replacing Skype for Business with Mattermost.

Out-of-Band Incident Response for Microsoft-Centric Environments
-----------------------------------------------------------------

During high-stakes incidents, Microsoft 365 tools can be limited or unavailable, slowing down response times and jeopardizing mission continuity.

**Benefits**

- **Maintain operational continuity during M365 outages** with a dedicated, out-of-band Mattermost instance for secure incident response, communication, and collaboration. See :doc:`Playbooks in Microsoft Teams </integrate/playbooks-for-microsoft-teams>`
- **Accelerate responses** with :ref:`AI-powered workflows <configure/enable-copilot:mattermost configuration>`, enabling structured playbooks for triage, escalation, and resolution even when primary systems are compromised.

> - **Integrate with Microsoft Security Suite** and :doc:`Entra ID </onboard/sso-entraid>` to preserve centralized identity management while keeping sensitive data in a secure secondary system.

Reviewer: Should we add a link to the integration capabilities like webhooks & slash commands?

> - **Protect breach-sensitive notifications** using :ref:`ID-only push alerts <configure/environment-configuration-settings:id-only push notifications>` and encrypted mobility, enabling secure communication without cloud exposure.

Reviewer: ```suggestion
- **Protect breach-sensitive notifications** using :ref:`ID-only push alerts <configure/environment-configuration-settings:id-only push notifications>` and enhanced mobile security, enabling secure communication without cloud exposure.
```

Reviewer: Let's also add a link to our Mobile Security Guide in the edit.

External Collaboration with Full Control
------------------------------------------

Managing external collaboration within Microsoft Teams can be complex, often requiring numerous configurations and administration that lead to security risks like usage of consumer-grade chat tools.

**Benefits**

> - **Integrate Mattermost with Microsoft Teams and Outlook** to enable secure external collaboration with encryption, audit trails, and role-based permissions—without compromising compliance. (See :doc:`Microsoft Teams Sync </collaborate/collaborate-within-connected-microsoft-teams>`).

Reviewer: I think we want this to be the M365 mission collab solution

- **Eliminate shadow IT** by providing :doc:`Connected Workspaces </onboard/connected-workspaces>` for sanctioned, policy-enforced engagement with external partners—reducing reliance on consumer-grade tools.

> - **Apply granular policy enforcement for external users**, including :doc:`read-only roles </onboard/delegated-granular-administration>`, :doc:`legal hold </comply/legal-hold>`, :doc:`retention policies </comply/data-retention-policy>`, and :doc:`custom Terms of Service </comply/custom-terms-of-service>`.

Reviewer: Let's swap out the "read-only roles" for "granular user permissions" with link to channel permission/moderation docs

- **Synchronize user identity** using :doc:`Entra ID </onboard/sso-entraid>` to maintain scalable, centralized access control across both internal and external collaborators.

Sovereign Communication in Microsoft Teams  
-------------------------------------------

Agencies and critical infrastructure organizations must often comply with strict data sovereignty rules that restrict cloud usage for sensitive collaboration.

**Benefits**

> - **Deploy Mattermost on-premise or in sovereign clouds**, fully integrated with Microsoft Teams and Outlook (See :doc:`Join Microsoft Team Meetings </integrate/microsoft-teams-meetings>`) to maintain workflow continuity and secure data storage.

Reviewer: This was also about the embedded mission 365 experience.

- **Store messages, recordings, and transcriptions in compliance-approved systems**, with :ref:`data-at-rest encryption <about/security:data-at-rest encryption>` ensuring no leakage of sensitive data to third-party platforms.
- **Enable secure Microsoft Teams interactions via embedded Mattermost collaboration**, supporting operations within familiar interfaces while enforcing regulatory compliance. See :doc:`Mattermost for M365, Teams, and Outlook </integrate/mattermost-mission-collaboration-for-m365>`.
- **Enforce agency-specific policies** with :doc:`legal hold </comply/legal-hold>`, :doc:`retention policies </comply/data-retention-policy>`, and :doc:`user access controls </onboard/advanced-permissions>` that align with national or sectoral mandates.

Cross-Instance Collaboration Hub
---------------------------------

Multi-agency, multi-tenant Microsoft 365 environments often hinder seamless collaboration and increase complexity for inter-organization workflows.

**Benefits**

> - **Centralize communication across M365 instances** using Mattermost as a neutral, embedded hub for messaging, file sharing, and playbook coordination (See :doc:`Playbooks in Microsoft Teams </integrate/playbooks-for-microsoft-teams>`).

Reviewer: M365 mission collab instead of MS Team Playbooks

===================================

Unlock the full potential of your Microsoft Teams, M365, and Entra ID investment with Mattermost. Designed for operational teams that require advanced customization, secure workflows, and seamless deployment on segregated networks, Mattermost enhances your existing Microsoft solutions for critical mission success.  

Mattermost complements Microsoft solutions with tailored capabilities designed to meet the unique needs of high-security environments, inter-agency collaboration, and external workflows, enabling teams to maximize their Microsoft investment.  

Use cases
----------

On-premises Skype for Business replacement
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

As Skype for Business is phased out, secure teams need a new communication solution that meets stringent air-gap and security standards. Mattermost operates entirely within your infrastructure, trusted in air-gapped environments, and integrates smoothly with Outlook and Azure AI to maintain secure operations while preserving full data control.

- **Seamless Microsoft integrations**: Integrates with Outlook and Azure AI to maintain secure workflows while preserving data sovereignty.

> - **Customizable deployment**: A trusted solution for replacing Skype for Business as it is phased out, ensuring compliance and operational success.

Reviewer: ```suggestion
- **Flexible deployment**: Ensuring compliance and operational success with self-hosted deployment in air-gapped, hyberscaler private cloud, gov Cloud or VM deployment options. 
```

- :doc:`Microsoft Calendar Sync </integrate/microsoft-calendar>`: Keep your team synced with calendar updates from Microsoft M365 directly in Mattermost, streamlining scheduling and meeting management.
- :doc:`Join Microsoft Team Meetings </integrate/microsoft-teams-meetings>`: Launch and manage video and audio calls directly from Microsoft Teams within Mattermost for unified, secure communication.
- :doc:`Playbooks in Microsoft Teams </integrate/playbooks-for-microsoft-teams>`: Access active, repeatable processes and real-time status updates directly in Microsoft Teams, improving cross-organizational alignment and operational efficiency.
- :doc:`Microsoft Teams Sync </collaborate/collaborate-within-connected-microsoft-teams>`: Securely sync chats and group messages between Microsoft Teams and Mattermost for real-time collaboration and streamlined communication.
- :doc:`Community for Mattermost </guides/community-for-mattermost>`: Engage with the Mattermost community directly within M365, Outlook, or Teams for support, insights, and collaboration on future product developments.

> Take control of your collaboration tools today and see how Mattermost complements and enhances your Microsoft infrastructure for seamless, secure, and scalable operations. Try Mattermost now and transform the way your teams collaborate.

Reviewer: Page below had "get started" this one doesn't. Let's be consistent.
